Sources/NIOPosix/NonBlockingFileIO.swift

```diff
@@ -712,3 +730,356 @@ public struct NIODirectoryEntry: Hashable {
     }
 }
 #endif
+
+@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+extension NonBlockingFileIO {
+    /// Read a `FileRegion` in ``NonBlockingFileIO``'s private thread pool which is separate from Task pool.
```

Comment:
I find the `which is separate from Task pool` part to be confusing in all of those doc comments. Can we just drop this?

Reply:
Yeah it was originally about running outside of EventLoops. I'm not happy with the wording either.